packages/documentation/src/pages/platform/tools/downtime-notifications.mdx

```diff
@@ -111,6 +111,27 @@ The render-flow for this container is:
     - If the current time is in the timeframe of the downtime window, render an alert banner informing the user that this application is undergoing maintenance.
     - Otherwise, `DowntimeNotification` will renders its children, in this case `MyAppDataGrid`. A dismissible modal is also rendered if downtime is approaching within the hour.
 
+### Adding a new PagerDuty service
+If you've created a new backend service and have a react application dependent on it, follow these steps to allow the react application to fetch it's downtime windows:
+
+#### Configuration
+- Login to PagerDuty (https://dsva.pagerduty.com/)
+- Create your PagerDuty **Service** (https://dsva.pagerduty.com/service-directory)
+  - This may require you to create a PagerDuty Person, Escalation Policy, and/or Team that is specific to this service.
+  - Follow the naming convention of other backend services in PagerDuty by prepending "External: " to the service name (i.e. "External: MyService").
+  - You may also want to create multiple services for multiple enviornements (i.e. "External: MyService", "Staging: External: MyService", "Dev: External: MyService").
+- In the [vets-api]() repo, add your service to the list of [maintenance services](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/settings.yml#L441) in `config/settings.yml`.
+- In the [devops]() repo, make the same configuration change to each enviornemnt's config file.
+- Deploy all changes.
+
+#### Testing
+- Login to PagerDuty (https://dsva.pagerduty.com/)
+- Go to your **staging** service
+- Create a maintenance window sometime in the future.
+  - You can specify a message to a user by using the token "USER_MESSAGE: " in the description (i.e. "Down for v16 update. USER_MESSAGE: We'll be back soon!")
```

Comment:
I vaguely know of this mechanism, but I don't know if it's really supported. Might also be an issue without sign-off from content writers. Could be worth noting.

Reply:
I'm not even sure if the UI reads that, but I confirmed that the message does come through on the API response. My intention is only to document what exists and how it works, not provide instructions on what you **should** do. But since it exists, I just documented it.

Reply:
Added a note here to notify the reader that there are additional considerations.